ext/sockets/conversions.c

```diff
@@ -365,8 +363,7 @@ void from_zval_write_int(const zval *arr_value, char *field, ser_context *ctx)
 	}
```

Comment:
These `has_error` checks are no longer going to work after some of your changes.